python/ray/train/v2/tests/test_trainer.py

```diff
@@ -305,6 +313,66 @@ def train_func():
         assert len(predictions) == 3
 
 
+class TestTensorflowTrainer:
+    @pytest.mark.parametrize("num_workers", [1, 2])
+    def test_tensorflow_linear(self, ray_start_4_cpus, num_workers):
+        def train_func(config):
+            result = tensorflow_linear_train_func(config)
+            assert len(result) == epochs
+            assert result[-1]["loss"] < result[0]["loss"]
+
+        num_workers = num_workers
+        epochs = 3
+        scaling_config = DataParallelScalingConfig(num_workers=num_workers)
+        config = {
+            "lr": 1e-3,
+            "batch_size": 32,
+            "epochs": epochs,
+            "dataset_name": TRAIN_DATASET_KEY,
```

Comment:
Do we still need this now that the example is directly calling `train.get_train_dataset_shard()`?

Reply:
nope, good catch